src/receiver.mjs

```diff
@@ -0,0 +1,57 @@
+
+import AudioBackend from './backend/index.mjs'
+import receiveProcessor from './processors/receive.mjs'
+
+const getUserAudio = async ( ) =>
+    navigator.mediaDevices.getUserMedia( {
+        audio: {
+            echoCancellation: false
+        }
+    } )
```

Comment:
Is echo cancellation disabling a requirement for quietjs to work reliably?

Reply:
Not sure, this was the old implementation so i respected that (sounded valid also)